### PR TITLE
Add Pontoon URL to README

### PR DESCRIPTION
**Pull request by pdehaan**
Words are not great. Suggestions/commits welcome!

r? @flodolo

### CODE REVIEW

**Comment by pdehaan** [comment]
Looks a bit clumsy, but we could also do a custom "Pontoon" badge-esque thing at the top of the README.md file:

editing_send_readme_md_at_master_ _mozilla_send

```md
[![Firefox Send on Pontoon](https://img.shields.io/badge/Pontoon-L20n-7bc876.svg)](https://pontoon.mozilla.org/projects/test-pilot-firefox-send/)
```

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ $ redis-server /usr/local/etc/redis.conf
 
 ## Localization
 
-_Coming soon_ (see [#57](https://github.com/mozilla/send/issues/57))
+Firefox Send localizations are managed via [Pontoon](https://pontoon.mozilla.org/projects/test-pilot-firefox-send/).
```
Comment (flodolo) [inline README.md]: I have a personal preference for "localization is managed", but plural works too. The badge is nice, but not sure it's effective (you would need to know what Pontoon or l20n are).

Maybe worth adding a bit more of information:

```
Firefox Send localization is managed via [Pontoon](https://pontoon.mozilla.org/projects/test-pilot-firefox-send/), not direct pull requests to the repository. If you want to fix a typo, add a new language, or simply know more about localization, please get in touch with the [existing localization team](https://pontoon.mozilla.org/teams/) for your language, or Mozilla’s [l10n-drivers](https://wiki.mozilla.org/L10n:Mozilla_Team#Mozilla_Corporation) for guidance.
```

**Comment by pdehaan** [comment]
@flodolo Yeah, yours is _way_ better, thanks! Pushing new commit.

**Comment by pdehaan** [comment]
OK, I think I got all the current/active Test Pilot experiments (except [Tab Center](https://testpilot.firefox.com/experiments/tab-center), which has been retired), as well as Screenshots and Activity Stream. I'll leave it to the repo owners to merge as they see fit.

**Pull request merged**